src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/XamlRequestDispatcherFactory.cs

```diff
@@ -91,10 +88,42 @@ private class XamlRequestDispatcher : RequestDispatcher
         }
     }
 
+    [ExportLspServiceFactory(typeof(LspWorkspaceManager), StringConstants.XamlLspLanguagesContract), Shared]
+    internal class XamlLspWorkspaceManagerFactory : LspWorkspaceManagerFactory
```

Comment:
💡 Need to enable the one-type-per-file analyzer

Reply:
yeah these should probably all be split out.  will update!